launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -25,9 +25,9 @@
 
 import static org.apache.spark.launcher.CommandBuilderUtils.*;
 
-/**
+/** 
  * Launcher for Spark applications.
- * <p/>
```

Comment:
Was this because some version of javadoc rejects it? I feel like it might. In theory this is supposed to be used like `<p>...</p>` always, so wrap the preceding line with these to create a line break after?